index.md

```diff
@@ -53,8 +53,9 @@ status: editors-draft
   <p>Combining accessibility standards and usability processes with real people ensures that web design is technically and functionally usable by people with disabilities. This is referred to as <em>usable accessibility</em> or <em>accessible user experience (UX)</em>.</p>
 
   <h3 id="usable-accessibility">Usable Accessibility</h3>
-  <p>Web designers and developers can use usability processes, methods, and techniques, such as user-centered design (UCD) process and user experience design, to address the user interface component of accessibility. While the considerations of people with disabilities are not always included in common practices, they can be easily integrated into existing usability processes, methods, and techniques.</p>
-  <p class="listintro">A key aspect is incorporating <strong>real people</strong> in design, including:</p>
+  <p>Web designers and developers can use usability, or user experience design processes, methods, and techniques, to address the user interface component of accessibility. While the considerations of people with disabilities are not always included in common practices, they can be easily integrated into user experience design.
```

Comment:
@James-Green I like this simplification of the sentence. Yet it dropped "user-centered design (UCD)", which could be a helpful keyword. Also, currently UCD and UX are sub-topics of "usability". In the new text usability and UX seem to be separate aspects. Were these intentional changes or something that just needs editorial tweaking?